client/jetbrains/src/main/java/com/sourcegraph/cody/chat/ResetCurrentConversationAction.java

```diff
@@ -0,0 +1,31 @@
+package com.sourcegraph.cody.chat;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.components.ServiceManager;
+import com.intellij.openapi.project.DumbAwareAction;
+import com.intellij.openapi.project.Project;
+import com.sourcegraph.cody.UpdatableChat;
+import com.sourcegraph.cody.UpdatableChatHolderService;
+import com.sourcegraph.common.ErrorNotification;
+import org.jetbrains.annotations.NotNull;
+
+public class ResetCurrentConversationAction extends DumbAwareAction {
+  @Override
+  public void actionPerformed(@NotNull AnActionEvent e) {
+    Project project = e.getProject();
+    if (project == null) {
+      displayUnableToResetConversationError();
+      return;
+    }
+    UpdatableChatHolderService updatableChatHolderService =
+        ServiceManager.getService(project, UpdatableChatHolderService.class);
+    UpdatableChat updatableChat = updatableChatHolderService.getUpdatableChat();
+    updatableChat.resetConversation();
+  }
+
+  private static void displayUnableToResetConversationError() {
+    ErrorNotification.show(
+        null,
+        "Unable to reset the current conversation with cody. Please try again. If you are seeing this error again and again, please reach out to support@sourcegraph.com.");
```

Comment:
- Capital "C" for Cody
- Make error message more in line with [this guide](https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/actionable_language/#error-messages)

```suggestion
        "Unable to reset the current conversation with Cody. Please try again or reach out to us at support@sourcegraph.com.");
```

Reply:
Thank you for clearing that out, I wasn't sure about that message, should have told you about that beforehand. And thanks for the link to the docs, interesting stuff